Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1452,6 +1453,7 @@ - (BOOL)materializeAndCacheObject:(id)object
         }];
         if (image == nil) {
             image = [PINImage pin_decodedImageWithData:container.data skipDecodeIfPossible:skipDecode];
+            image = [PINImage pin_scaledImageForImage:image withKey:key];
```

Comment:
This won't work for processed images I don't think as the key will be an combination of the URL and a processing key. Either we need a method to extract the URL from the key or we need to pass the URL down to this method :(

See `cacheKeyForURL:processorKey:`

All though the burden added by supporting processing images is huge, sometimes I wonder if we should kill it…